src/swap/SwapScreen.tsx

```diff
@@ -397,7 +461,7 @@ export function SwapScreen({ route }: Props) {
     }
   }
 
-  const allowReview = useMemo(
+  const allowSwap = useMemo(
     () => Object.values(parsedSwapAmount).every((amount) => amount.gt(0)) && !fetchingSwapQuote,
```

Comment:
should we add a check for if the quote is "possible" if using viem?

Reply:
I think not: because if the quote is not "possible" it should allow to press the button, but instead of starting swap it will display a bottom sheet